packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -577,10 +577,12 @@ export const audiusBackend = ({
     return {
       ...track,
       // TODO: This method should be renamed as it does more than images.
-      duration: track.track_segments.reduce(
-        (duration, segment) => duration + parseFloat(segment.duration),
-        0
-      ),
+      duration:
+        track.duration ||
```

Comment:
Use ?? instead of || cause 0 is technically falsy

Reply:
I think we want to do this when falsey because the discovery migration sets `duration` to 0 for every track. I'm not sure if we should even support a track actually being 0 seconds, but that would break this (could fix by optional chaining the track_segments and fall back to 0)

Reply:
Oh yeah great call! Let's do this

Reply:
sweet 🚀 just realized I misspoke re: edge case where a track is actually 0 seconds - `track_segments` will never be null (it'll be at worst an empty array), so no need for optional chaining. this will just return 0 if someone for some reason uploads an empty track (lol)